README.md

```diff
@@ -72,6 +72,13 @@ For example:
 ./bin/logs-example --config log.level=debug
 ```
 
+Option can also be set by environment variable by prefix the environmental
+variable with `AUTOGRADER__` and assign it the desired value. 
```

Comment:
"This information is not complete, the user does not have the information on what to do for multi-word keys."

Same comment as the last round!